### PR TITLE
Docs update: move safegraph to inactive category

### LINKED ISSUE #500: Safegraph mobility signals are dead; modify API docs
```
Social Distancing Metrics v2.1 (formerly Physical Distancing Metrics)	
*Please note that this dataset is no longer being updated as of 4/19/21. We recommend 
the weekly patterns datasets in its place for continued research.*Aggregated daily views
of USA foot-traffic summarizing movement between census block groups.
```

We might be able to bring up a different set of signals based on the weekly patterns dataset, but the existing signals are definitely toast.

Modify the safegraph API docs listing the whole suite of mobility signals as permanently inactive.

### PR DESCRIPTION
**Pull request by dshemetov**
Fixes #500.

### CODE REVIEW

**Comment by chinandrew** [comment]
safegraph patterns is still up while the mobility ones arent, so only a subset of the signals are being retired

**Comment by dshemetov** [comment]
ah, i'll fix that

**Comment by dshemetov** [comment]
Fixed. By the way, on [this](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/indicator-combination.html) page, the link to the inactive combo signals indicator is broken, but I'm not sure why. It's formatted correctly in the markdown, but the docs maker doesn't replace the page .md with html 🤷

File: docs/api/covidcast-signals/safegraph-inactive.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* **Source name:** `safegraph`
+* **Available for:** county, MSA, HRR, state (see [geography coding docs](../covidcast_geography.md))
+* **Time type:** day (see [date format docs](../covidcast_times.md))
+* **License:** [CC BY](../covidcast_licensing.md#creative-commons-attribution)
+
+This data source uses data reported by [SafeGraph](https://www.safegraph.com/)
+using anonymized location data from mobile phones. SafeGraph provides several
+different datasets to eligible researchers. We surface signals from two such
+datasets.
+
```
Comment (krivard) [inline docs/api/covidcast-signals/safegraph-inactive.md:18]: Need a blurb here on when it was cancelled

**Review by krivard — APPROVED** [review]
👍  🚀

**Pull request merged**